### ISSUE TITLE
Does Not Honor "Do Not Disturb" option

### ISSUE THREAD

**Issue by amiga-500** [httpdispatch/MissedNotificationsReminder#49]
Android has a "Do Not Disturb" option for apps.
This app should have a feature where you can have it NOT alert when "Don Not Disturb" is enabled AND an override option to still play when "Do Not" is enabled.  

This is common in other apps.  For example, for AquaMail, under "App Notification" you will see a "Message Notifications" area and inside there is an "Override Do Not Disturb" which is disabled.  This app also has a "Priority Notifications" which has the same options but you can enable the "Override Do Not Disturb" so it can alert you when certain emails come in...Excellent Idea.

Sadly, it seems this app is abandoned :(

**Comment by httpdispatch** [comment]
Such feature is already present in the advanced settings

**Comment by soilentgreen** [comment]
Yes the feature is already in advanced setting. But it don't work. I still get reminder when I'm switch on Do Not Disturb.
Can it be fix?
Thanks.
